packages/babel-generator/test/fixtures/misc/.placeholders/input.js

```diff
@@ -0,0 +1,8 @@
+var %%a%% = %%b%%
+
+%%c%%
+
+class %%d%% {}
+class A %%e%%
+
+function %%f%%(...%%g%%) %%h%%
```

Comment:
None of the tests added a semicolon. But there should be cases that do, if I look at the code above?

Reply:
The semicolon has been added after the variable (but it has been inserted by the `VariableDeclaration` printer) and after the statement placeholder (by the placeholder printer).
It hasn't been added after function/class declaration because they are never followed by a semicolon.

Reply:
Oppss I looked at the wrong file.